hazelcast/src/test/java/com/hazelcast/test/HazelcastTestSupport.java

```diff
@@ -105,6 +105,11 @@ public static void assertUtilityConstructor(Class clazz) {
         }
     }
 
+    // overridden in another context.
+    protected Config getConfig() {
```

Comment:
Rename to createConfig. Get config makes it sounds like your are getting a config from somewhere, but actually you are creating a new one. Create is in line with the other creational methods in this class.